math/src/main/scala/breeze/linalg/NumericOps.scala

```diff
@@ -25,6 +25,8 @@ import scala.reflect.ClassTag
 
 
 trait ImmutableNumericOps[+This] extends Any {
+
+  //ToDo: Could this be renamed to be more informative? Kind of sounds like Python repr()
   def repr: This
 
```

Comment:
representation. The scala collections library uses this too in TraversableLike, XLike, etc.